src/components/WeeklySummary/WeeklySummary.jsx

```diff
@@ -241,6 +256,21 @@ export class WeeklySummary extends Component {
     this.setState({ errors: errors || {} });
     if (errors) return;
 
+    let currentSubmittedCount = 0;
+    if (this.state.formElements.summary !== '') {
+      currentSubmittedCount += 1;
```

Comment:
You can update the state here and your logic is fine. just add `currentSubmittedCount` as a state and update it using `this.setState`